src/ScriptEngine/ScriptEngine.cs

```diff
@@ -27,40 +27,58 @@ public class ScriptEngine : BaseUnityPlugin
 
         ConfigEntry<bool> LoadOnStart { get; set; }
         ConfigEntry<KeyboardShortcut> ReloadKey { get; set; }
+        
+        // FS Watcher EDIT
+        ConfigEntry<bool> QuietMode { get; set; }
+        ConfigEntry<bool> EnableFileSystemWatcher { get; set; }
+        ConfigEntry<bool> IncludeSubdirectories { get; set; }
+        
+        private FileSystemWatcher fileSystemWatcher;
+        private bool shouldReload;
+        //
 
         void Awake()
         {
             LoadOnStart = Config.Bind("General", "LoadOnStart", false, new ConfigDescription("Load all plugins from the scripts folder when starting the application"));
             ReloadKey = Config.Bind("General", "ReloadKey", new KeyboardShortcut(KeyCode.F6), new ConfigDescription("Press this key to reload all the plugins from the scripts folder"));
+            QuietMode = Config.Bind("General", "QuietMode", false, new ConfigDescription("Suppress sending log messages to console except for the error ones."));
+            EnableFileSystemWatcher = Config.Bind("General", "EnableFileSystemWatcher", false, new ConfigDescription("Listens for changes to the scripts directory and reloads all plugins automatically."));
```

Comment:
This config option is defined but doesn't seem to be used anywhere. I assume it should be used to enable/disable the call to `StartFileSystemWatcher()`?

Reply:
Sorry, that was a big mistake, luckily you saw it in time. It's fixed now.

Thanks @ghorsington!